core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/progressmessage/ProgressMessageSilverpeasV5.java

```diff
@@ -27,9 +27,11 @@
 import org.apache.ecs.ElementContainer;
 import org.apache.ecs.xhtml.div;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.ecs.xhtml.div'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)